### ISSUE TITLE
Breaking change `specs` option for v2.0

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#228]
## Currently: 

https://markuplint.dev/configuration#specs

```json
{
  "specs": ["@markuplint/vue-spec", "./path/to/custom-specs/any-lang"]
}
```

**It has a problem that applied the specs to all langs and extensions.**

## Changes:

```json
{
  "specs": {
    ".vue$": "@markuplint/vue-spec",
    "any-regex": "./path/to/custom-specs/any-lang"
  }
}
```

In v2.0, Settable by array but apply it to all langs. It is deprecated. It will be obsolete in v3.0.

**Comment by YusukeHirao** [comment]
Resolved on #237

**Comment by YusukeHirao** [comment]
- [ ] Rewrite about it to the docs.
- [ ] Fix schema files.
